source/16-6-2_Outcomes.markdown

```diff
@@ -70,15 +104,73 @@ If you have more than one measure to add to a category, you can also opt to put
 - Click on 'bulk add new {{outcome}} measures.
 - Enter the measures in the text box, each one separated by a semi-colon. If you choose to add them in this way, you'll need to find each one in the list and edit it (click on the pen and paper icon to the right to open the editing box) to include the type, values and notes, as we saw with the single measure before. 
 
-[Editing Icon](16.6.2f.png)
+![Editing Icon](16.6.2f.png)
 
 - Continue these steps until you have added all the measures that you need.
 
+#### Scores with user specified issue
+
+This measure type is intended to be used where you want to record a score, but also want to record what the issue is.
+One option is to create a set of different measures, one for each type of issue, and recording the detail in the 
+comments box. However this can feel prescriptive and may not be appropriate for your situation. It can also lead to 
+a long list that becomes hard to navigate.
+
+The "scores with user specified issue" measure type allows you to record the score and the issue in one place. The 
+first time you enter data for a particular {{user}} you will be prompted to enter the issue. The next time you enter
+data for that {{user}} you will see the previously entered issue and can just update the score.
+
+A downside of this is that it means that when reporting in aggregate you will be combining different types of 
+issue. One person may make progress in self-confidence while another in their housing situation: when reporting 
+overall you will see that two people made some progress on 'Issue 1'. Using separate measures would allow you to see 
+the progress on each issue separately.
+
+Another limitation is that you can only enter this data through the particular tab for the {{outcome}} category in 
+their profile. This is because each person's issue will be different.  So when setting up these measures, you will 
+need to add the additional tab to profiles in 
+[personal settings](https://lamplight.online/en/admin/settings/what/allsettings) -
+see [16.4 Personal settings](/help/index/p/16.4) for help with this.
+
+
+
+### Data entry types for different measures
+
+Information on how to add and update descriptors is given in the sections below. This explains how your choices of 
+the measure will affect the data entry options once descriptors are set up.
```

Comment:
**the** measure will affect the data entry options once descriptors are set up.
**DELETE**
the measure will affect the data entry options once **the** descriptors are set up.
**ADD**